docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-monitoring-stack.adoc

```diff
@@ -30,13 +28,12 @@ controller-2.redhat.local   192.168.24.10  mon _admin mgr
 6 hosts in cluster
 ----
 +
-Confirm that the cluster is healthy and both `ceph orch ls` and
+Confirm that the cluster is healthy and that both `ceph orch ls` and
 `ceph orch ps` return the expected number of deployed daemons.
```

Comment:
Can we add an example of a ceph orch ps command?

Reply:
@fmount Were you able to find an example?

Reply:
I wasn't able to find an available deployment with the monitoring stack.
I think I can take a note and follow up on this point, so I don't block this patch.